.github/workflows/pull-request.yml

```
@@ -201,23 +194,39 @@
         if: steps.changed-files.outputs.only_changed != 'true'
-        run: npm run test -- --watchAll=false --coverage
+        env:
+          NODE_V8_COVERAGE: './.nyc_output'
+        run: |
+          npm run test:vitest:coverage
 
+      - name: Merge Coverage Reports
+        if: steps.changed-files.outputs.only_changed != 'true'
+        run: |
+          npx nyc merge
+          mv coverage.json ./coverage/lcov.info     
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)